docs/user-guide/configmap.md

```diff
@@ -2,6 +2,11 @@
 
 ---
 
+The configuration options are listed below. To specify which ConfigMap to use, pass the `configmap` parameter to `nginx-ingress-controller`. For example,
```

Comment:
I think it would be better to show this in https://github.com/kubernetes/ingress-nginx/blob/master/deploy/README.md since many users will deploy using this guide it would be helpful to show how it is created in https://github.com/kubernetes/ingress-nginx/blob/master/deploy/configmap.yaml and pointed to in https://github.com/kubernetes/ingress-nginx/blob/master/deploy/without-rbac.yaml#L25.

Reply:
Sounds good. I'll make the change.